15/umbraco-cms/tutorials/starter-kit/lessons/install-the-starter-kit.md

```diff
@@ -0,0 +1,65 @@
+# Install the Starter Kit
+
+Installing the Starter Kit provides a pre-built set of templates, content types, and demo content to explore or kickstart your Umbraco project.
+
+## Outcome
+
+You’ll have the Umbraco Starter Kit installed in your local project, ready to explore in the backoffice. This setup adds example content and templates — perfect for learning or quick prototyping.
```

Comment:
Let's try to avoid "hyphens" in sentences.

```suggestion
You’ll have the Umbraco Starter Kit installed in your local project, ready to explore in the backoffice. This setup adds example content and templates, perfect for learning or quick prototyping.
```